sherpa/tests/test_stack.py

```diff
@@ -17,22 +17,77 @@
 #  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
+import os
+
+import pytest
+
+from sherpa.utils.testing import requires_stk
 
-from sherpa.utils.testing import SherpaTestCase, requires_stk
 
-import os
 _this_dir = os.path.dirname(__file__)
 
 
+def get_name(name):
+    return '/'.join((_this_dir, name))
```

Comment:
```suggestion
    return os.path.join(_this_dir, name)
```
will work on windows, too, where the separator is "\".

Reply:
[we have many places where we'd need to change things to get it to work on windows...] Also, it's not clear what the stack library would do on Windows so I'm not going to change this (i.e. this is testing the stk library and not a generic path).